lib/kitchen/verifier/inspec.rb

```diff
@@ -174,7 +174,7 @@ def runner_options_for_winrm(config_data)
           'host' => URI(kitchen[:endpoint]).hostname,
           'port' => URI(kitchen[:endpoint]).port,
           'user' => kitchen[:user],
-          'password' => kitchen[:pass],
+          'password' => kitchen[:password],
```

Comment:
Should we make a fallback to `kitchen[:pass]`, this would allow us to use kitchen-inspec with older and newer versions?